tools/add-fpr-rule

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+from argparse import ArgumentParser
+import os
+import sys
+
+sys.path.append('/usr/share/archivematica/dashboard')
+os.environ["DJANGO_SETTINGS_MODULE"] = "settings.local"
+
+from django.db import connection
+
+from fpr.models import IDCommand, IDRule, IDTool, Format, FormatGroup, FormatVersion, FPCommand, FPRule, FPTool
+
+FORMAT_FIELDS = (
+    "description",
+    "pronom_id",
+    "version",
+)
+
+IDCOMMAND_FIELDS = (
+    "description",
+    "config",
+    "replaces_id",
+    "script",
+    "script_type",
+    "tool_id",
+)
+
+IDRULE_FIELDS = (
+    "command_output",
+    "command_id",
+    "format_id",
+)
+
+IDTOOL_FIELDS = (
+    "description",
+    "version",
+)
+
+FPCOMMAND_FIELDS = (
+    "description",
+    "tool_id",
+    "command_usage",
+    "command",
+    "script_type",
+    "output_location",
+    "output_format_id",
+    "verification_command_id",
+    "event_detail_command_id",
+    "replaces_id",
+)
+
+
+FPRULE_FIELDS = (
+    "purpose",
+    "command_id",
+    "format_id",
+)
+
+FPTOOL_FIELDS = (
+    "description",
+    "version",
+)
```

Comment:
These can be generated automatically from ```[f.column for f in <model>._meta.fields]```.  You'd have to strip ```uuid```, ```lastmodified``` and ```enabled``` from the list though.  I'm concerned about this getting out of date.  Though, this is a devtool, so it's not as big a deal if these lists are out of date.